apps/trading/components/navbar/navbar.tsx

```diff
@@ -1,21 +1,25 @@
+import classNames from 'classnames';
 import { useRouter } from 'next/router';
-import { Vega } from '../icons/vega';
 import Link from 'next/link';
+import { NetworkSwitcher } from '@vegaprotocol/environment';
 import { t } from '@vegaprotocol/react-helpers';
-import classNames from 'classnames';
+import { Vega } from '../icons/vega';
 import { useGlobalStore } from '../../stores/global';
 
 export const Navbar = () => {
   const { marketId } = useGlobalStore();
   const tradingPath = marketId ? `/markets/${marketId}` : '/';
   return (
     <nav className="flex items-center">
-      <Link href="/" passHref={true}>
-        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-        <a className="px-[26px]">
-          <Vega className="fill-white" />
-        </a>
-      </Link>
+      <div className="flex items-center h-full">
+        <Link href="/" passHref={true}>
+          {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+          <a className="px-[26px]">
```

Comment:
Suggestion (non-blocking): absolutely fine if 26px exactly is needed, but if not, you could use either 24px or 28px from the theme

Reply:
absolutely, changed this to 24

Reply:
I think Sam meant to use `p-24` which is set up in our spacing definitions in the theme

Reply:
oh, i see, i didn't get that part. applied another fix.